TicTacToe2/TicTacToe/Board.cs

```diff
@@ -6,5 +6,74 @@ namespace TicTacToe2.TicTacToe
 {
     class Board
     {
+        private char[,] space; //Board
+        private int gridNumber = 0; //Board
+
+        private void ChooseBoard() //Board
+        {
+            Console.WriteLine("\nThis grid is user defined.");
+            Console.WriteLine("ie.Inputing 3 will result in a grid 3 rows by 3 columns.");
+            gridNumber = GetUserInput(3, 1000);
```

Comment:
Rather than getting this value in the board class, it would make more sense thematically to pass the board's size as an argument to its constructor.

https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/constructors